config-service-change-event-generator/src/main/java/org/hypertrace/config/service/change/event/api/ConfigChangeEventGenerator.java

```diff
@@ -0,0 +1,46 @@
+package org.hypertrace.config.service.change.event.api;
+
+import com.google.protobuf.Value;
+import org.hypertrace.core.grpcutils.context.RequestContext;
+
+/** The interface config change event generator. */
+public interface ConfigChangeEventGenerator {
+
+  /**
+   * Send create notification for newly added config
+   *
+   * @param requestContext requestContext
+   * @param configType config type
+   * @param context context
+   * @param config newly created config
+   */
+  void sendCreateNotification(
```

Comment:
can we create overloads without a context field, since that's optional for configs (better than explicitly passing nulls, which is I'm guessing what's required right now)

Reply:
Done